src/dos/cdrom_image.cpp

```diff
@@ -457,12 +479,14 @@ bool CDROM_Interface_Image::PlayAudioSector(unsigned long start, unsigned long l
 	if (relative_start < 0) {
 		len -= relative_start;
 	}
+
 	// Seek to the calculated byte offset, bounded to the valid byte offsets
-	const int offset = (track->skip +
-	                    + clamp(relative_start, 0, track->length - 1)
+	const int offset = (track->skip
+	                    + clamp(relative_start, 0, static_cast<signed>(track->length - 1))
```

Comment:
`signed` is the same as `int`, I think?

Reply:
<int> it is now; and matches offset's `int` value type.